hazelcast/src/main/java/com/hazelcast/internal/cluster/impl/operations/MemberRemoveOperation.java

```diff
@@ -18,15 +18,17 @@
 
 import com.hazelcast.internal.cluster.impl.ClusterDataSerializerHook;
 import com.hazelcast.internal.cluster.impl.ClusterServiceImpl;
+import com.hazelcast.internal.cluster.impl.MembershipManagerCompat;
 import com.hazelcast.logging.ILogger;
 import com.hazelcast.nio.Address;
 import com.hazelcast.nio.ObjectDataInput;
 import com.hazelcast.nio.ObjectDataOutput;
-import com.hazelcast.spi.impl.AllowedDuringPassiveState;
 
 import java.io.IOException;
 
-public class MemberRemoveOperation extends AbstractClusterOperation implements AllowedDuringPassiveState {
+@Deprecated
+// not used on 3.9+
```

Comment:
We already do it in `com.hazelcast.internal.cluster.impl.MembershipManagerCompat#removeMember()` which is called within this operation.